models/skipthoughts.py

```diff
@@ -0,0 +1,194 @@
+"""
+A simple model based on skipthoughts sentence embeddings.
```

Comment:
Also, can you think of some single-sentence summary of what the skipthoughts model does? Like "A previously proposed model based on bidi-RNN-with-memory trained to predict preceding and followup words", is that remotely correct? My memory is a bit hazy.